syft/frameworks/torch/torch_attributes.py

```diff
@@ -238,3 +238,21 @@ def is_inplace_method(self, method_name):
             is_inplace = method_name[-1] == "_" and "__" not in method_name
             self.inplace_methods[method_name] = is_inplace
             return is_inplace
+
+    @staticmethod
+    def apply_fix16922(torch):
```

Comment:
Should this be here?

Reply:
@midokura-silvia

Reply:
yes, the code using torch.jit does not work with torch version 1.1. While we are waiting for the fix to be released, we need to keep the fixes for torch 1.0.1.

Reply:
Can we update the comment to be in relation to PyTorch 1.1.1 rather than PyTorch 1.0.2 then?

Reply:
+1 to changing it to 1.1.1.